content/en/docs/components/katib/experiment.md

```diff
@@ -41,159 +42,166 @@ the experiment from the UI or from the command line.
-    The template 
-    uses the [Go template format](https://golang.org/pkg/text/template/).
-    
-    You can define the job in raw string format or you can use a 
-    [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/). 
-    [Here](https://github.com/kubeflow/katib/blob/master/manifests/v1alpha3/katib-controller/trialTemplateConfigmapLabeled.yaml) is an example how to create ConfigMap with trial templates.
-
-* **parallelTrialCount**: The maximum number of hyperparameter sets that Katib
-  should train in parallel.
-
-* **maxTrialCount**: The maximum number of trials to run.
+- **parallelTrialCount**: The maximum number of parallel trials to run.
+  This is equivalent to the number of hyperparameter sets that Katib should train in parallel.
+
+- **maxTrialCount**: The maximum number of trials to run.
   This is equivalent to the number of hyperparameter sets that Katib should
   generate to test the model.
 
-* **maxFailedTrialCount**: The maximum number of failed trials before Katib
+- **maxFailedTrialCount**: The maximum number of failed trials before Katib
   should stop the experiment.
-  This is equivalent to the number of failed hyperparameter sets that Katib 
+  This is equivalent to the number of failed hyperparameter sets that Katib
   should test.
   If the number of failed trials exceeds `maxFailedTrialCount`, Katib stops the
   experiment with a status of `Failed`.
 
-* **metricsCollectorSpec**: A specification of how to collect the metrics from
+- **trialTemplate**: The template that defines the trial.
+  You must package your ML training code into a Docker image, as described
+  [above](#docker-image). You must configure the model's
+  hyperparameters either as command-line arguments or as environment variables,
+  so that Katib can automatically set the values in each trial.
+
+  You can use one of the following job types to train your model:
+
+  - [Kubernetes Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+    (does not support distributed execution).
+  - [Kubeflow TFJob](/docs/components/training/tftraining/) (supports
+    distributed execution).
+  - [Kubeflow PyTorchJob](/docs/components/training/pytorch/) (supports
+    distributed execution).
+
+  See the [`TrialTemplate`
+  type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1beta1/experiment_types.go#L201-L261).
+  The template
+  uses the [unstructured format](https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured).
+
+  You can define the job in YAML format or you can use a
+  [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/).
+  [Here](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/katib-controller/trial-template-configmap.yaml)
+  is an ConfigMap example with trial templates.
```

Comment:
```suggestion
  For trial templates, check [this example of a ConfigMap](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/katib-controller/trial-template-configmap.yaml)
```

Reply:
I will refactor this section in the following PR with doc for: https://github.com/kubeflow/katib/issues/1341, https://github.com/kubeflow/katib/issues/1280.